recipes/biorbd-casadi/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set libname = "biorbd" %}
+{% set name = "biorbd-casadi" %}
+{% set version = "1.2.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pyomeca/{{ libname }}/archive/Release_{{ version }}.tar.gz
+  sha256: 7682fa0c34d6c9a4415c8293d12089845390516845334dbb5c4fcaedb60872ef
+
+build:
+  number: 0
+  skip: true  # [win]
+  skip: true  # [py<36 or py>37]
+  rpaths:
+    - lib/
+    - lib/{{ libname }}/
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja
+    - swig
+  host:
+    - casadi
+    - rbdl-casadi
+  run:
+    - casadi
```

Comment:
Both `casadi` and `rbdl-casadi` look like they provide shared libraries but aren't using `run_exports` (see above). This should probably be fixed in the upstream recipes before this is merged.

If there is a reason not to it should use `pin_compatible` (see the numpy comment above).

Reply:
I will fix that in other recipes. If I have a problem, can I ping you directly from these repo?

Reply:
Go ahead :)